02.calendar/cal.rb

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env ruby
+
+require 'optparse'
+require 'date'
+
+params = ARGV.getopts("", "y:#{Date.today.year}", "m:#{Date.today.month}")
+month = params["m"].to_i
+year = params["y"].to_i
+
+start_date = Date.new(year,month,1) 
+end_date = Date.new(year,month,-1)  
+day_of_week = start_date.wday
+
+wdays = ["日 " + "月 " + "火 " + "水 " + "木 " + "金 " + "土 "]
+
+  puts "#{month}月 #{year}".center(20)
+  puts wdays
+  space_per_day = 3
+  print " " * space_per_day * day_of_week
```

Comment:
```suggestion
  SPACE_PER_DAY = 3
  print " " * SPACE_PER_DAY * day_of_week
```

常に固定の値は定数として定義しましょう。
そもそもここは

```suggestion
  SPACE_PER_DAY = "   "
  print SPACE_PER_DAY * day_of_week
```

のようにしても良いと思います

Reply:
定数を表現するために`space_per_day`を`SPACE_PER_DAY`に変更しました！
また、スペースは"   "で表現しました！